service/flannelconfig/v2/resource_set.go

```diff
@@ -198,6 +198,11 @@ func NewResourceSet(config ResourceSetConfig) (*framework.ResourceSet, error) {
 		if key.VersionBundleVersion(customObject) == VersionBundle().Version {
 			return true
 		}
+		// TODO remove this hack with the next version bundle version or as soon as
+		// all certconfigs obtain a real version bundle version.
+		if key.VersionBundleVersion(customObject) == "" {
+			return true
+		}
```

Comment:
Note I added this here like we have in cert-operator. This is apparently missing and thus we do not reconcile the flannel magics in old clusters.